code/modules/surgery/organs/tails.dm

```diff
@@ -121,3 +121,57 @@
 	if(istype(tail_owner))
 		tail_owner.dna.species.mutant_bodyparts -= "tail_monkey"
 		tail_owner.update_body()
+
+// Lazy copy paste from felinid code
+/obj/item/organ/tail/fox
+	name = "fox tail"
+	desc = "A severed fox tail. Sad."
+	tail_type = "Fox"
+
+/obj/item/organ/tail/fox/Insert(mob/living/carbon/human/tail_owner, special = FALSE, drop_if_replaced = TRUE)
+	..()
+	if(istype(tail_owner))
+		var/default_part = tail_owner.dna.species.mutant_bodyparts["tail_human"]
+		if(!default_part || default_part == "None")
+			if(tail_type)
+				tail_owner.dna.features["tail_human"] = tail_owner.dna.species.mutant_bodyparts["tail_human"] = tail_type
+				tail_owner.dna.update_uf_block(DNA_HUMAN_TAIL_BLOCK)
+			else
+				tail_owner.dna.species.mutant_bodyparts["tail_human"] = tail_owner.dna.features["tail_human"]
+			tail_owner.update_body()
+
+/obj/item/organ/tail/fox/Remove(mob/living/carbon/human/tail_owner, special = FALSE)
+	..()
+	if(istype(tail_owner))
+		tail_owner.dna.species.mutant_bodyparts -= "tail_human"
+		color = tail_owner.hair_color
+		tail_owner.update_body()
+
+/obj/item/organ/tail/fox/alt
+	name = "fox tail"
+	desc = "A severed fox tail. Sad."
+	tail_type = "Fox 2"
+
+/obj/item/organ/tail/shark
+	name = "fox tail"
+	desc = "A severed fox tail. Sad."
```

Comment:
Are these supposed to be fox tails?

Reply:
Damn it, forgot to change it. I'll update it